examples/gradle-kt-dsl/app/src/main/java/com/amazon/corretto/crypto/examples/AesXtsBenchmark.java

```diff
@@ -0,0 +1,110 @@
+package com.amazon.corretto.crypto.examples;
+
+import com.amazon.corretto.crypto.provider.AmazonCorrettoCryptoProvider;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.infra.Blackhole;
+import org.openjdk.jmh.profile.GCProfiler;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+import javax.crypto.Cipher;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+import java.security.SecureRandom;
+import java.util.concurrent.TimeUnit;
+
+@Warmup(iterations = 4, time = 4)
+@Measurement(iterations = 4, time = 4)
+@Fork(value = 1)
+public class AesXtsBenchmark {
```

Comment:
can we add this to our [existing benchmarking suite](https://github.com/corretto/amazon-corretto-crypto-provider/tree/main/src/jmh/java/com/amazon/corretto/crypto/provider)? if this file needs to live in `examples/`, perhaps we can symlink it to `src/jmh/java/com/amazon/corretto/crypto/provider`?

Reply:
Done